docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -73,11 +72,37 @@ Installing roles
   - extracting example.role to /home/user/.ansible/roles/example.role
   - example.role was installed successfully
 
-
 * List all installed roles:
 
 .. code-block:: bash
 
   ansible-galaxy role list
 
 See :ref:`ansible-galaxy` for detailed documentation.
+
+ansible
+=======
+
+Running ad-hoc commands
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* Install a package
+
+.. code-block:: bash
+
+  ansible localhost -m ansible.builtin.apt -a "name=apache2 state=present" -b -K
+
+Runs  ``ansible localhost``- from your local system
+- ``name=apache2 state=present`` - this installs the package `apache2` on a Debian based system.
+- ``-b`` - executes with elevated privileges (uses :ref:`become <become>`).
+- ``become`` - executes with root privileges.
+- ``-m`` - module name.
+- ``-K`` - prompts to ask for privilege escalation password (sudo password).
```

Comment:
```suggestion
- ``-K`` prompts for the privilege escalation password.
```

Reply:
I'm not sure it's 100% always the case this is the root user password. I'd just keep it consistent with the general docs: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_privilege_escalation.html#become-command-line-options